packages/web-components/src/components/va-text-input/test/va-text-input.e2e.ts

```diff
@@ -355,7 +355,7 @@ describe('va-text-input', () => {
     expect(await inputEl.getProperty('autocomplete')).toBe('email');
   });
 
-  // Begin USWDS v3 test
+  // Begin USWDS tests
```

Comment:
I noticed that the va-text-input component did not have all of the existing tests added for USWDS so I'm adding them to this PR but if we would prefer this submitted as a separate PR, I can do that.